### PR TITLE
Add types alpha

### PR DESCRIPTION
**Pull request by NullableInt**
#336 but with base as main
@Xillians this is your playground, contact me if help is needed.
edit: I can not add reviewers or assignees, if you could help me out here @asbjornu that would be greatly appreciated! 🙏

### CODE REVIEW

**Review by cobraz — COMMENTED** [review]
AFAIK, this are all the types for the full API?

**Comment by NullableInt** [comment]
> AFAIK, these are all the types for the full API?

Yes, I've ported all types from the .Net SDK. I have still to fix the build error.

File: package.json [hunk 1]
```diff
@@ -46,6 +46,8 @@
   "author": "Bjerk AS <office@bjerk.io>",
```
Comment (asbjornu) [inline package.json:46]:
```suggestion
  "author": "Swedbank Pay <opensource@swedbankpay.com>",
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Review by Xillians — APPROVED** [review]

**Comment by NullableInt** [comment]
Looks like we are getting close to something great!
@asbjornu should approve, perhaps we should bribe him?

**Comment by asbjornu** [comment]
Where do I collect my bribe?

**Comment by NullableInt** [comment]
> Where do I collect my bribe?

Have your guys connect with my guys, and we'll have it delivered at the usual drop of point.

**Pull request merged**